### DISCUSSION TITLE
Tag workflows being triggered when new branch is created

### DISCUSSION THREAD

**Discussion by fahanson** [community/community#26229] [Actions]
<p>We have seperate workflows for when a new branch is created, and when we create a new tag. Currently whenever a new branch is created in Github, both the branch and the tag workflows are run. I have tried a lot of different iterations of this, but cannot for the life of me get this working. No matter what I try, when a new branch is created the tag workflow is run (as well as the branch, which obviously is fine).</p>
<p>Here are the iterations I have tried (all of them run when a new branch is created):</p>
<pre><code>name: tag workflow

on:
  create:
    tags:

name: tag workflow

on:
  push:
    branches-ignore:
      - '*'
    tags:
      - 'v-*'
  create:
    branches-ignore:
      - '*'
    tags:
      - 'v-*'

name: tag workflow

on:
  push:
    branches-ignore:
      - '*'
    tags:
  create:
    branches-ignore:
      - '*'
    tags:

name: tag workflow

on:
  push:
    branches-ignore:
    tags:
  create:
    tags:
</code></pre>
<p>To reiterate, I only want this workflow to run when a new tag is created not when a new branch is created.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/fahanson">@fahanson</a> ,</p>
<p>The yaml syntax is incorrect in your workflow. The ‘create’ event will be triggered when a new branch or a tag created. Tags/branches-ignore are used for push/pull_request not for create event. Please check official doc below for more details:</p>
<p><a href="https://help.github.com/en/actions/reference/events-that-trigger-workflows#create-event-create" rel="nofollow noopener">https://help.github.com/en/actions/reference/events-that-trigger-workflows#create-event-create</a></p>
<p><a href="https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags" rel="nofollow noopener">https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags</a></p>
<p>If you’d like to only monitor the new tag, you should remove ‘create’ event in yaml file, code sample as below:</p>
<pre><code>on:
  push:
    branches-ignore:
      - '*'
    tags:
      - 'v*'
</code></pre>
<p>Thanks.</p>
